test/integration/controllermanager/shoot/maintenance/utils_test.go

```diff
@@ -40,6 +41,33 @@ func waitForShootToBeMaintained(shoot *gardencorev1beta1.Shoot) {
 	}).Should(BeFalse())
 }
 
+func waitMachineImageVersionToBeExpiredInCloudProfile(cloudProfileName, imageName, imageVersion string, expirationDate *metav1.Time) {
+	Eventually(func(g Gomega) {
```

Comment:
Should this be `EventuallyWithOffset` as described in https://github.com/gardener/gardener/blob/master/docs/development/testing.md#writing-tests-relevant-for-all-kinds:

> use `ExpectWithOffset` for making assertions in helper funcs like `expectSomethingWasCreated`: [example test](https://github.com/gardener/gardener/blob/2eb54485231408cbdbabaa49812572a07124364f/extensions/pkg/controller/controlplane/genericactuator/actuator_test.go#L732-L736)

(same below)

Reply:
Good point, thank you!